tradeexecutor/state/trade.py

```diff
@@ -278,6 +278,26 @@ def __post_init__(self):
     def strategy_cycle_at(self):
         """Alias for oepned_at"""
         return self.opened_at
+    
+    @property
+    def fee_tier(self) -> (float | int):
```

Comment:
It's not good practice to have float | int, because it does not make any sense.

Please make sure fee tier is always float.

Please document what kind of fee tiers can be returned.